executor/Makefile

```diff
@@ -57,13 +57,13 @@ add_protos:
 test: fmt vet
 	go test ./api/... ./predictor/... ./k8s/...  -coverprofile cover.out
 
-copy-openapi-resources:
-	mkdir -p openapi/
-	cp ../openapi/swagger-ui/* openapi/
-	cp ../openapi/engine.oas3.json openapi/seldon.json
+copy_openapi_resources:
+	mkdir -p api/rest/openapi/
+	cp ../openapi/swagger-ui/* api/rest/openapi/
```

Comment:
I think it would be best if we left the UI bits outside the Go source code, at the root `./executor` level. Just thinking in case we also want to expose the gRPC using Swagger UI (not sure if that's possible though).

Reply:
Unfortunately for testing it has to be relateive to the rest folder, and then when executor is built it becomes a single executable, so that ensures the path is consistent